archunit/src/test/java/com/tngtech/archunit/library/dependencies/testexamples/cyclewithunbalanceddependencies/CycleWithUnbalancedDependenciesRoot.java

```diff
@@ -0,0 +1,4 @@
+package com.tngtech.archunit.library.dependencies.testexamples.cyclewithunbalanceddependencies;
+
```

Comment:
A brief note about the need of this class (making the tests refactoring safe) would be helpful.

Reply:
Yeah, you're right, I've added it analogously to the others... :+1: